OSDOCS-5637-console: adds last 2 console bugs to RNs (#59614)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

* Previously, when downloading a `kubeconfig` file for a ServiceAccount, an error was displayed and the ServiceAccount token was unable to be reached. This error was due to the removal of automatically generated secrets. With this update, the download `kubeconfig` action has been removed and the error no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-7308[*BZ#OCPBUGS-7308*])

> * In the {product-title} 4.12 release, the node terminal tab would display an error because of missing annotations that were caused by pod security measures. Without the required annotations, the node debug pod cannot start. With this update, {product-title} adds these annotations, so that the node debug pod can start and the node terminal tab loads without any errors. (link:https://issues.redhat.com/browse/OCPBUGS-4252)

Reviewer: A couple suggestions and a few console terminology items
```suggestion
* Previously, the *Terminal* tab on the *Node details* page displayed an error because of missing annotations that were caused by pod security measures. Without the required annotations, the node debug pod cannot start. With this update, {product-title} adds these annotations, so the node debug pod can start and the *Terminal* tab loads without any errors. (link:https://issues.redhat.com/browse/OCPBUGS-4252)
```

> * Previously, if a user with regular account privileges attempted to issue the `oc delete csv` command when uninstalling the Operator, the Operator's subscription becomes stuck. After this action, an administrator cannot reinstall the uninstalled Operator, because a conflict exists with the stuck subscription. With this update, a detailed error message displays when an administrator attempts to reinstall the uninstalled Operator. (link:https://issues.redhat.com/browse/OCPBUGS-3822[*OCPBUGS-3822])

Reviewer: I tweaked this a bit, up to you what to take or leave. I will say "user with regular account privledges" should be "cluster admin"

```suggestion
* Previously, if a cluster administrator attempted to issue the `oc delete csv` command when uninstalling the Operator, the Operator's subscription becomes stuck. The administrator was unable to reinstall the Operator because a conflict existed with the subscription. With this update, a detailed error message displays when an administrator attempts to reinstall the uninstalled Operator. (link:https://issues.redhat.com/browse/OCPBUGS-3822[*OCPBUGS-3822])
```
